pkgs/development/libraries/libspatialite/default.nix

```diff
@@ -49,13 +49,15 @@ stdenv.mkDerivation rec {
   '';
 
   # Failed tests (linux & darwin):
-  # - check_extension
-  # - check_sql_stmt_extension
   # - check_virtualtable6
   # - check_drop_rename
-  # - check_get_normal_zoom_extension_load
   doCheck = false;
 
+  preCheck = ''
+    export LD_LIBRARY_PATH=$(pwd)/src/.libs
+    export DYLD_LIBRARY_PATH=$(pwd)/src/.libs
+  '';
```

Comment:
Does this signal an intention to enable the tests?

Reply:
Yes, in the future, currently there are 2 failed tests:
```
libspatialite> FAIL: check_virtualtable6
libspatialite> =========================
libspatialite> Unexpected num of rows for test 28: 0.
libspatialite> FAIL: check_drop_rename
libspatialite> =======================
libspatialite> RenameTable "table_1" error: RenameTable exception - error in trigger ISO_metadata_reference_row_id_value_insert: no such column: rowid.
```
I don't think that's something nix specific.

Reply:
Wonder if it's expecting a specific sqlite version.